app/containers/App/tests/index.test.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 
+import ProgressBar from 'components/ProgressBar';
 import Header from 'components/Header';
 import Footer from 'components/Footer';
 import App from '../index';
```

Comment:
This is getting weird. Might take some time to figure this one out. I'm on it...